src/NativeModules/UIManager.js

```diff
@@ -61,6 +61,11 @@ const UIManager = {
   configureNextLayoutAnimation(callback, errorCallback) {
 
   },
+  AndroidDrawerLayout: {
+    Constants: {
+      DrawerPosition: { Left: 8388611, Right: 8388613 },
```

Comment:
Yes, but it would be better if they were accurate. Well, if it's the values they use, it's good enough for me!